tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -114,7 +114,7 @@ def test_zos_job_name_query_multi_wildcards_func(ansible_zos_module):
 
 def test_zos_job_id_query_short_ids_func(ansible_zos_module):
     hosts = ansible_zos_module
-    qresults = hosts.all.zos_job_query(job_id="STC003")
+    qresults = hosts.all.zos_job_query(job_id="STC00002")
```

Comment:
The problem is that neither `STC003` or any short ID is available in our ECs anymore. We have `STC00002` but not `STC002`. This test is the only one that tests for a specific ID, instead of using a wildcard, so even if we're not using an ID that's less than 8 chars, we're still testing that we can get the output for a single job.

I shouldn't hard code the ID, ideally, but that's outside the scope of the backport and Demetri was going to create a new issue so we can instead submit our own job with a short ID, wait, and then query it, avoiding this issue altogether.

Reply:
Never mind, now the original job is back and we can query not only `STC00003`, but `STC003` too. I can change it back